### PR TITLE
manager: Make matching of executables more accurate

### PR DESCRIPTION
**Pull request by johan-bjareholt**
- [x] Test with system modules
- [x] Test with bundled modules
- [ ] Test on windows

### CODE REVIEW

**Review by ErikBjare — COMMENTED** [review]
Looks good, just fix the pathsep assumptions.

File: aw_qt/manager.py [hunk 1]
```diff
@@ -88,7 +87,11 @@ def _discover_modules_system() -> List["Module"]:
     modules: List["Module"] = []
     paths = [p for p in search_paths if os.path.isdir(p)]
     for path in paths:
-        for filename in _filter_filenames(os.listdir(path)):
+        for filename in os.listdir(path):
+            if not filename.startswith("aw-"):
+                continue
+            if not is_executable(path+"/"+filename):
```
[the remark this answers is not in the document]
Comment (johan-bjareholt) [reply aw_qt/manager.py]: Cool, would be even better though if you could amend the old commit so there either are not multiple "fix: X" commits or a single "fix: fixed multiple things" commit.

[the remark this answers is not in the document]
Comment (ErikBjare) [reply aw_qt/manager.py]: One could always squash at the end.

**Comment by johan-bjareholt** [comment]
@ErikBjare I've tested bundled modules now too, only needs to be tested on windows now.

**Review by ErikBjare — APPROVED** [review]
Looks good, will test on Windows.

Edit: Actually I won't, but I'll test before making a release.

**Pull request merged**